verma2021explaining/journal/metadata.yaml

```diff
@@ -5,46 +5,56 @@
 #  - For a successful replication, it should be prefixed with "[Re]"
 #  - For a failed replication, it should be prefixed with "[¬Re]"
 #  - For other article types, no instruction (but please, not too long)
-title: "ReScience (R)evolution"
+title: "[Re] Explaining Groups of Points in Low-Dimensional Representations"
 
 # List of authors with name, orcid number, email and affiliation
 # Affiliation "*" means contact author (required even for single-authored papers)
 authors:
-  - name: Konrad Hinsen
-    orcid: 0000-0003-0330-9428
-    email: konrad.hinsen@cnrs.fr
-    affiliations: 1,2
+  - name: Rajeev Verma
+    orcid: 0000-0002-2340-0942
+    email: rajeev.verma@student.uva.nl
+    affiliations: 1*
```

Comment:
add a comma between 1 and `*` - this is why the yaml converter was failing as it could not find a contact name.